core-model/src/main/java/com/sequenceiq/cloudbreak/domain/stack/Stack.java

```diff
@@ -655,7 +655,7 @@ public StopRestrictionReason isInfrastructureStoppable() {
         if ("AWS".equals(cloudPlatform())) {
             for (InstanceGroup instanceGroup : instanceGroups) {
                 if (instanceGroup.getTemplate().getVolumeTemplates().stream()
-                        .filter(volume -> "ephemeral".equals(volume.getVolumeType())).findAny().isPresent()) {
+                        .anyMatch(volume -> "ephemeral".equals(volume.getVolumeType()))) {
```

Comment:
we should have `ephemeral` as a constant somewhere instead of a string multiple places

Reply:
is there any unit test for this?

Reply:
Yes, please check now.